code/game/objects/structures/crates_lockers/closets/secure/guncabinet.dm

```diff
@@ -51,3 +51,8 @@
 			. += "off"
 		else if(locked)
 			. += "locked"
+
+/obj/structure/closet/secure_closet/guncabinet/cc
+	req_access = list(ACCESS_CENT_SPECOPS_COMMANDER)
+	icon = 'icons/obj/guncabinet.dmi'  //if this isnt here, it uses the secure locker sprite. I dont know why, but it does and this is needed.
+	icon_state = "base" // same as above
```

Comment:
test again without these, it probably only happened because you broke that other path.

Reply:
fixed in latest!